app/main/db/queries.py

```diff
@@ -54,69 +59,130 @@ def fuzzy_search(query_string):
+        query = generate_consignment_reference_filter_query(consignment_id)
+    else:
+        query = generate_browse_everything_query()
+
+    query_results = None
+    try:
+        query_results = db.session.execute(query)
     except exc.SQLAlchemyError as e:
         print("Failed to return results from database with error : " + str(e))
+
+    if query_results is not None:
+        results = process_results(
+            query_results, transferring_body_id, series_id, consignment_id
+        )
     return results
 
 
-def get_file_data_grouped_by_transferring_body_and_series():
+def get_user_accessible_transferring_bodies(access_token):
+    unique_transferring_bodies = []
+    bodies = None
+    try:
+        bodies = Body.query.all()
+    except exc.SQLAlchemyError as e:
+        print("Failed to return results from database with error : " + str(e))
+
+    # get transferring bodies for which user has access to
+    user_transferring_body_groups = get_user_transferring_body_groups(
+        access_token
+    )
+
+    if bodies is not None and user_transferring_body_groups is not None:
+        for body in bodies:
+            group_name = body.Name
+            if len(user_transferring_body_groups) > 0:
+                for user_group in user_transferring_body_groups:
+                    if (
+                        user_group.strip().replace(" ", "").lower()
+                        == group_name.strip().replace(" ", "").lower()
+                    ):
+                        unique_transferring_bodies.append(group_name)
+    return unique_transferring_bodies
+
+
+def get_file_meta_data(file_id):
     results = []
+    query = (
+        db.select(FileMetadata.PropertyName, FileMetadata.Value)
+        .join(File, FileMetadata.FileId == File.FileId)
+        .where((func.lower(File.FileType) == "file") & (File.FileId == file_id))
+    )
+    query_results = None
+    try:
+        query_results = db.session.execute(query)
+    except exc.SQLAlchemyError as e:
+        print("Failed to return results from database with error : " + str(e))
+
+    if query_results is not None:
+        for r in query_results:
+            record = {"PropertyName": r.PropertyName, "PropertyValue": r.Value}
+            results.append(record)
+    return results
 
+
+def generate_browse_everything_query():
     query = (
         db.select(
-            Body.Name.label("transferring_body"),
-            Series.Name.label("series"),
+            Body.BodyId.label("BodyId"),
+            Body.Name.label("TransferringBody"),
+            Series.SeriesId.label("SeriesId"),
+            Series.Name.label("Series"),
             func.max(Consignment.TransferCompleteDatetime).label(
-                "last_record_transferred"
+                "Last_Record_Transferred"
             ),
             func.count(func.distinct(Consignment.ConsignmentReference)).label(
-                "consignment_in_series"
+                "Consignment_in_Series"
```

Comment:
can we make all the labels throughout this PR snake case?

Or at least can we make them consistently CamelCase if its going to be weird to have model fields as CamelCase and labels as snake case?

```suggestion
                "consignment_in_series"
```

Reply:
fixed issues with dictionary to refer snake case variable names